maisim/maisim.Game/Beatmaps/Beatmap.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace maisim.Game.Beatmaps
+{
+    /// <summary>
+    /// Class that represents a beatmap that includes all the information needed on beatmap.
+    /// </summary>
+    public class Beatmap
+    {
+        // TODO: Implement SongSet when it's available
+
+        public Beatmap(TrackMetadata trackMetadata, DifficultyLevel difficultyLevel, float difficultyRating, bool isRemaster, float maxSeasonalScore, string noteDesigner)
```

Comment:
You can yeet the constructor as well since properties can be initialized at object creation. Same applies for other db models

Reply:
Yeah changed and it's look better